util/config/test/test_superusermanager.py

```
@@ -1,8 +1,10 @@
 from dataclasses import dataclass
-from test.fixtures import *
+from unittest.mock import MagicMock, patch
 
 import pytest
 
+from data.users import FederatedUserManager
+from test.fixtures import *
```

Ruff (0.14.14)

[error] `from test.fixtures import *` used; unable to detect undefined names

(F403)